canonicalwebteam/store_api/stores/snapstore.py

```diff
@@ -169,3 +169,24 @@ def update_store_snaps(self, session, store_id, snaps):
         )
 
         return self.process_response(response)
+
+    def get_store_invites(self, session, store_id):
+        headers = self._get_authorization_header(session)
+
+        response = self.session.get(
+            url=self.get_endpoint_url(f"{store_id}/invites"),
```

Comment:
I can't see a `GET` endpoint for `/invites`, looking at their docs this information might be already available with `get_store`

Reply:
`get_store` just has:

```python
{
    'name': 'Web and Design', 
    'roles': ['admin', 'review', 'view', 'access'], 
    'id': 'ahnuP3quahti9vis8aiw'
}
```